### DISCUSSION TITLE
Addition of a context menu to the chat tabs?

### DISCUSSION THREAD

**Discussion by papertek** [ppy/osu#15300] [Ideas] [answered]
Would be cool if you were to right-click users and channels and see a little context tab sorta like google chrome:
![image](https://user-images.githubusercontent.com/72106503/138809696-23e24e5d-d600-46c0-b454-d3db5501234f.png)

**Comment by bdach** [comment, marked as answer]
There was a pull request adding something like that already but it was blocked on there not being an efficient API endpoint to close all channels.

Also note that there's a pending redesign for chat.

**Reply by papertek** [reply]
👍
